resources/github-comment-markdown.template

```diff
@@ -57,9 +57,11 @@
     ${(testsSummary?.failed > 10) ? '> Show only the first 10 test failures' : ''}
     <% testsErrors?.findAll{item -> item?.status == "FAILED"}?.take(10)?.each { test -> %>
 * **Name**: `${test?.name}`
-  * Age: ${test?.age}
-  * Duration: ${test?.duration}
-  * Error Details: ${test?.errorDetails}
+<% age = "* Age: ${test?.age}"%>
+<% duration = (test?.duration >= 0 ) ? "(took ${Math.round(test.duration/1000/60)} min ${Math.round(test.duration/1000)%60} sec)" : ''%>
+<% errorDetails = (test?.errorDetails?.trim() && !test?.errorDetails.equals('null')) ? "\n  * Error Details: ${test?.errorDetails}" : ''%>
+<% errorStackTrace = (test?.errorStackTrace?.trim() && !test?.errorStackTrace.equals('null')) ? "\n  * Error Stacktrace: ${test?.errorStackTrace}" : ''%>
+  ${age} ${duration} ${errorDetails} ${errorStackTrace}
```

Comment:
One single line will help to avoid empty spaces such as, the highlighted section where the errorDetails does not exist

![image](https://user-images.githubusercontent.com/2871786/97859138-11bd0700-1cf8-11eb-92c7-37e48720ed75.png)